src/middlewares/__tests__/ssm.js

```diff
@@ -189,6 +217,39 @@ describe('🔒 SSM Middleware', () => {
     })
   })
 
+  test(`It should call onChange along with aws-sdk if cache enabled but cached param has expired`, (done) => {
+    testScenario({
+      ssmMockResponse: {
+        Parameters: [{Name: '/dev/service_name/secure_param', Value: 'something-secure'}]
+      },
+      middlewareOptions: {
+        names: {
+          secureValue: '/dev/service_name/secure_param'
+        },
+        cache: true,
+        onChange: onChange,
+        cacheExpiryInMillis: 10,
+        setToContext: true,
+        paramsLoaded: false
+      },
+      callbacks: [
+        (_, {context}) => {
+          expect(onChange.mock.calls.length).toEqual(1)
```

Comment:
Just FYI there is [.toHaveBeenCalledTimes(number)](https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber) API:)

Reply:
I knew there must be something like this but couldn't find it - changed :)